ui/src/lib/api-client/fms-client.ts

```diff
@@ -15,3 +19,66 @@ const authMiddleware: Middleware = {
 	}
 };
 fmsClient.use(authMiddleware);
+
+// Our swagger specs have some parameters defined as query parameters, but also include those parameters as
+// part of the path. Those path parameters cannot be replaced correctly, so we will need to strip them out.
+const fixPathsMiddleware: Middleware = {
+	async onRequest({ request }) {
+		// Strip any path parameters that haven't been provided
+		// E.g. /teamIssues/{noteId} becomes /teamIssues
+		let fixedUrl = request.url.replace(/\/%7B\w*%7D/g, '');
+		return new Request(fixedUrl, request);
+	}
+};
+fmsClient.use(fixPathsMiddleware);
+
+export type TeamIssue = components['schemas']['TeamIssueModel'];
+export interface FetchTeamIssueOptions {
+	noteId?: string;
+	teamNumber?: number;
+	issueType?: string;
+	resolutionStatus?: string;
+}
+export async function getTeamNotes(
+	fetch: any,
+	options: {
+		noteId?: string;
+		teamNumber?: number;
+		issueType?: string;
+		resolutionStatus?: string;
+	}
```

Comment:
Should this be using the interface type defined above?

Reply:
Still getting used to Typescript and figuring the patterns out - I ended up not using the interface